packages/pluggableWidgets/accordion-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "accordion-native",
   "widgetName": "Accordion",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
I don't think this is breaking, as per the previous comment. But please check with WC.